eng/ci/host.coldstart.yml

```diff
@@ -13,6 +13,35 @@ schedules:
     always: true
 
 parameters:
+
+# Function apps to include in the run.
+- name: includeHelloHttpNet9
+  displayName: Include HelloHttpNet9 (.NET9 Web Application)
+  type: boolean
+  default: true
+- name: includeHelloHttpNet9NoProxy
+  displayName: Include HelloHttpNet9NoProxy (.NET9 Worker Application)
+  type: boolean
+  default: true
+- name: includeHelloHttpNode
+  displayName: Include HelloHttpNode (Node Application)
+  type: boolean
+  default: false
+
+# Dotnet trace profile collection & jit trace generation.
+- name: collectNetTrace
+  displayName: Collect DotNet trace
+  type: boolean
+  default: false
+- name: netTraceCollectArguments
+  displayName: DotNet trace profile collection arguments (Valid only if "Collect DotNet trace" is true)). Example `-n Microsoft.Azure.WebJobs.Script.WebHost --profile cpu-sampling --duration 00:00:00:10`
+  type: string
+  default: '-n Microsoft.Azure.WebJobs.Script.WebHost --providers Microsoft-Diagnostics-DiagnosticSource:0xfffffffffffff7ff:5,Microsoft-Windows-DotNETRuntime:0x10:5,Microsoft-Windows-DotNETRuntime:0x4000080018:5,FunctionsSystemLogsEventSource:0xFFFFFFFFFFFFFFFF:5 --stopping-event-provider-name Microsoft-Diagnostics-DiagnosticSource --stopping-event-event-name Activity1/Stop'
```

Comment:
>Microsoft-Windows-DotNETRuntime:0x10:5,Microsoft-Windows-DotNETRuntime:0x4000080018:5 [](http://example.com/codeflow?start=126&length=85)

are we adding Microsoft-Windows-DotNETRuntime twice?

Reply:
Consolidated it to `Microsoft-Windows-DotNETRuntime:0xC0001801F9`